core/src/main/scala/com/nvidia/spark/rapids/tool/profiling/Profiler.scala

```diff
@@ -595,4 +574,23 @@ object Profiler {
       propStr + s"\nComments:\n$commentsToStr\n"
     }
   }
+
+  /**
```

Comment:
Should this function be present in a common trait since its being used by both `Profiler` and `Qualification` now?

Reply:
Thanks @parthosa! Moving this function to common trait `RuntimeReporter`